tests/unit/backend/layers/api/test_portal_api.py

```diff
@@ -157,7 +150,7 @@ def test__get_collection_id__ok(self, mock_dt):
             "access_type": "WRITE",
             "contact_email": "john.doe@email.com",
             "contact_name": "john doe",
-            "created_at": 1234,
+            "created_at": mock.ANY,
```

Comment:
Probably a good idea. Mocking the timestamp isn't really the goal of this test 👍